src/components/List.vue

```diff
@@ -127,4 +104,28 @@ export default defineComponent({
 b {
   margin-left: 5px;
 }
+.shoulder {
```

Comment:
since these classes are referenced for the 3rd time, consider refactoring them into one, put them somewhere where it can be used everywhere without writing them again and again

Reply:
done